routing/pathfind.go

```diff
@@ -722,6 +722,13 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 		amountToSend := toNodeDist.netAmountReceived +
 			lnwire.MilliSatoshi(inboundFee)
 
+		// Check if accumulated fees would exceed fee limit when this
+		// node would be added to the path.
+		totalFee := int64(amountToSend) - int64(amt)
```

Comment:
I find this log helpful when checking the behavior from the itest,
```go
log.Tracef("Checking toNode(%v) with minInboundFee=%v, inboundFee=%v, amountToSend=%v, amt=%v, totalFee=%v", toNodeDist.node, minInboundFee, inboundFee, amountToSend, amt, totalFee)
```

Reply:
+1 for the trace log, would be helpful when debugging inbound fees.